docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -72,7 +74,7 @@ for more details.
 .. _dronecan_support_device_temperature:
 
 uavcan.equipment.device.Temperature (Data Type ID = 1110)
-##########################################################
```

Comment:
Good to add a note/warning here that this is only sent if you have the old style telemetry